.travis.yml

```diff
@@ -8,27 +8,17 @@ before_install:
 - ./miniconda.sh -b -p ./lib/miniconda
 - export PATH=$(pwd)/lib/miniconda/bin:$PATH
 - conda update --yes conda
+- wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-113.0.0-linux-x86_64.tar.gz -nv
+- tar zxvf google-cloud-sdk-113.0.0-linux-x86_64.tar.gz -C lib
+- export PATH=$(pwd)/lib/google-cloud-sdk/bin:$PATH
 install:
 - conda install --yes python=$TRAVIS_PYTHON_VERSION cryptography numpy scipy
-- pip install -r requirements.txt
-- pip install -r test_requirements.txt
-- pip install -e src/common -e src/scraper -e src/server -e src/topicmodeller -e src/learner -e src/orchestrator
+- tools/install_envs.sh
 - pip install coveralls
```

Comment:
What i wanted to do was to put in install_env what was not specific to .travis. ie: what you would need to run in local.

So I left here
-"conda install" which is a travis hack to get numpy/Scipy (we don't have conda in local, we install numpy in the "normal" way)
-"coveralls" which is specific to pushing coverage metric to covelall website.

Does is seem OK for you ? I can also make a script like "install_env_travis"